### DISCUSSION TITLE
Looking for an opensource android SDK

### DISCUSSION THREAD

**Discussion by mohaa7** [fingerprintjs/fingerprintjs#837] [General]
Is there any opensource android SDK for the opensource fingerprintjs version?

**Comment by makma** [comment]
Hey @mohaa7,
Yes, there is, please take a look at: https://github.com/fingerprintjs/fingerprintjs-android

Cheers!

**Comment by mohaa7** [comment]
Please update the [readme.md](https://github.com/fingerprintjs/fingerprintjs/blob/master/readme.md) file:

![Unstitled](https://user-images.githubusercontent.com/118692398/206870407-fe99d4d7-228b-40aa-a277-feb98bbce9df.png)

**Reply by makma** [reply]
Good point, will do, thanks @mohaa7
